### PR TITLE
Update MCPi errors

### PR DESCRIPTION
**Pull request by mobilegmYT**

### CODE REVIEW

**Comment by Botspot** [comment]
Have you seen [this](https://github.com/Botspot/pi-apps/issues/144#issuecomment-760292432)?

**Comment by TheBrokenRail** [comment]
```sh
wget https://jenkins.thebrokenrail.com/job/minecraft-pi-docker/job/master/lastSuccessfulBuild/artifact/out/deb/minecraft-pi-native_1.0.0.20201225.1901_all.deb
```
This is a very bad idea. This link specifies a timestamp that is only use din one build, but is specifying the latest build. As soon as my Jenkins runs another build, this link will break. Please either specify a build number, or don't hard-code the timestamp.

File: apps/Minecraft Pi/install [hunk 1]
```diff
@@ -14,14 +14,16 @@ fi
...
+
 sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 04EE7237B7D453EC
 sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 648ACFD622F3D138
 echo 'deb http://deb.debian.org/debian buster-backports main' | sudo tee -a /etc/apt/sources.list
 sudo apt update
 
 "${DIRECTORY}/pkg-install" "libseccomp2" "$(dirname "$0")" || exit 1
 
-sudo apt remove docker docker-engine docker.io containerd runc
+sudo apt remove docker docker-engine docker.io containerd runc -y
```
Comment (TheBrokenRail) [inline apps/Minecraft Pi/install:26]: ``apt`` doesn't have a stable CLI interface, use ``apt-get``.

File: apps/Minecraft Pi/uninstall [hunk 2]
```diff
@@ -7,5 +7,11 @@ function error {
...
-sudo apt-get remove -y minecraft-pi-native
-sudo apt-get autoremove -y
+"${DIRECTORY}/purge-installed" "$(dirname "$0")" || exit 1
+
+sudo apt remove minecraft-pi-native mcpil-r -y
+sudo apt autoremove -y
+
+sudo mv /etc/apt/sources.list.bak /etc/apt/sources.list
+sudo rm /etc/apt/sources.list.d/Alvarito050506_mcpi-devs.list
+sudo apt update
```
Comment (TheBrokenRail) [inline apps/Minecraft Pi/uninstall:17]: ``apt`` doesn't have a stable CLI interface, use ``apt-get``.

**Comment by Botspot** [comment]
We'll use https://github.com/Botspot/pi-apps/issues/144.

**Pull request closed without merging**